structured_data_util_oof.php

```diff
@@ -18,7 +18,7 @@ public static function html2jsonld_oof($html)
 	    $jsonLd = null;
 	    $startScriptTagPos = strpos($html, '<script type="application/ld+json">');
 
-	    if ($startScriptTagPos) {
+	    if ($startScriptTagPos !== false) {
```

Comment:
This is a classic php one. I discovered that if the variable $startScriptTagPos is 0 (indicating that the substring is there but at position 0) that if condition will evaluate to false. In Most cases $startScriptTagPos was a positive integer which evaluated to true.

When we check the documentation :(https://www.php.net/manual/en/function.strpos.php) it suggests to use a false or not-false check which indeed works for both cases ( 0 + positive integer )